src/pages/join-us/JoinUs.tsx

```diff
@@ -17,19 +18,28 @@ export default function JoinUs(){
     };
 
     return(
-        <SubpageWrapper title={__('joinUsPage.header')}>
-            <>
-                {/*<button*/}
-                {/*    className={styles.button}*/}
-                {/*    onClick={() => window.open('https://docs.google.com/forms/d/e/1FAIpQLSf-aeTqPVKXVolgZyAsSVKhb2wuBDK5Izzq-5-eBxqQi-xKTw/viewform', '_blank')}*/}
-                {/*>*/}
-                {/*    {__('joinUsPage.form')}*/}
-                {/*</button>*/}
+        <>
+            <Helmet>
+                <title>{__('joinUsPage.meta.title')}</title>
+                <meta name="description" content={__('joinUsPage.meta.description')}/>
+                <meta property="og:title" content={__('joinUsPage.meta.title')}/>
+                <meta property="og:description" content={__('joinUsPage.meta.description')}/>
+                <meta property="og:image" content={String(require('../../assets/images/about-us-page/image2.png'))}/>
+            </Helmet>
+            <SubpageWrapper title={__('joinUsPage.header')}>
+                <>
+                    {/*<button*/}
```

Comment:
commented code could be deleted

Reply:
i left it so it would be easy to reenable it for next recruitments. i turned this functionality into a boolean at the top of the file to make it cleaner